sources/iTermPreferences.m

```diff
@@ -391,6 +395,14 @@ + (void)setInt:(int)value forKey:(NSString *)key {
     [self setObject:@(value) forKey:key];
 }
 
++ (NSUInteger)uintForKey:(NSString *)key {
+    return [(NSNumber *)[self objectForKey:key] unsignedIntegerValue];
```

Comment:
Does this do the right thing if a negative value was saved in an older version of iTerm2?